ingress-controller/scripts/apidocs-gen/generate.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 SCRIPT_ROOT="$(dirname "${BASH_SOURCE[0]}")/../.."
 CRD_REF_DOCS_BIN="$1"
 # Download the `kong/kubernetes-configuration` package and get its path.
-KUBE_CONF_REPO=github.com/kong/kubernetes-configuration
+KUBE_CONF_REPO=github.com/kong/kubernetes-configuration/v2
```

Comment:
I wonder why this came up only now and not before 🤔

Reply:
Maybe it's because we haven't regenerated the documentation before?

Reply:
Ah that's in KIC's subdir. Ok, then that's fine. 👍